.changeset/shy-carpets-own.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-org': major
```

Comment:
This should be a `patch` release.

Reply:
Could we also include an example of how we would adjust this limit? The changeset becomes part of the release notes so having that information is really helpful when people are upgrading

Reply:
Sure. Attaching video in PR description.